lib/msal-common/src/telemetry/performance/PerformanceClient.ts

```diff
@@ -440,18 +439,20 @@ export abstract class PerformanceClient implements IPerformanceClient {
 
     /**
      * Increment counters to be emitted when the measurements are flushed
-     * @param counters {Counters}
+     * @param fields {string[]}
      * @param correlationId {string} correlation identifier
      */
-    increment(counters: Counters, correlationId: string): void {
+    incrementFields(fields: { [key: string]: {} | undefined; }, correlationId: string): void {
```

Comment:
Should this be number? 

```suggestion
    incrementFields(fields: { [key: string]: number | undefined; }, correlationId: string): void {
```

Reply:
It can be a string, array or anything else.

Reply:
How do you increment a string? Or am I misunderstanding what this does?

Reply:
Sorry I thought you were referring to `add` method. Fixed it, thanks!